operations/pyroscope/helm/pyroscope/values.yaml

```diff
@@ -110,6 +110,9 @@ pyroscope:
 
   nodeSelector: {}
 
+  # -- Topology Spread Constraints
+  topologySpreadConstraints: [ ]
```

Comment:
looks like we hit a minor lint error here

Reply:
done